types/src/event.rs

```diff
@@ -131,6 +131,15 @@ impl TryFrom<&[u8]> for EventKey {
     }
 }
 
+impl TryFrom<String> for EventKey {
```

Comment:
Do we actually need this anywhere else? It seems we could easily do it in parse_* function() instead exposing this everywhere else, which might be confusing to other code, since String could also just hold bytes, in our case, it is  a specific format.

Reply:
It is not used anywhere yet. I think adding TryFrom<String> is convenient for the EventKey, 
This impl assumes the String is hex-encoded string, which is most likely case when we serialize bytes into string.
AccountAddress has similar func.

I added try_parse_from to combining this TryFrom<String> with deserialize for parsing EventKey, SignedTransaction and AccountAddress

Reply:
an better way is to create an enum::EventKeyParam that parse and returns EvenKey instead